### DISCUSSION TITLE
https://kdheepak.com/blog/the-basic-building-blocks-of-ratatui-part-1/

### DISCUSSION THREAD

**Discussion by giscus** [kdheepak/kdheepak.github.io#17] [Comments]
# https://kdheepak.com/blog/the-basic-building-blocks-of-ratatui-part-1/

https://kdheepak.com/blog/the-basic-building-blocks-of-ratatui-part-1/

**Comment by xav-ie** [comment]
Thank you for the article/tutorial! Excited for part II! :)
